.gitmodules

```diff
@@ -0,0 +1,11 @@
+[submodule "specifications/json-ld-api"]
```

Comment:
These are the specs, but since their sole purpose here is for testing, could this be named `test-suites/`?  That's what the scripts in jsonld.js use.

Reply:
I prefer the `specifications/` folder name, as I do hope/expect we'll get more unit tests added to `tests/` at some point, and `test-suites/` would certainly be confusing.